### PR TITLE
[main] Kogito-6694 : Assign user tasks to jdoe and alice in kogito-travel-agency-example

### PR DESCRIPTION
**Pull request by JudeNiroshan**
**NOTE! Please make sure to squash and merge with the following message!** 
> Kogito-6694 : Assign user tasks to jdoe and alice in extended/travels & extended/visas

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

https://issues.redhat.com/browse/KOGITO-6607

<details>
<summary>
Modified kogito-travel-agency-example/extended project. Both travels and visas project has been modified. The intention is to smoothly integrate kogito-task-console app into these projects. 
</summary>

- User tasks in travels business process definition assigned to user `jdoe`
- User task in visas business process definition assigned to user `alice`

### Reason:

Kogito-task-console app is only showing the tasks bounded to the logged-in user. Therefore, the user tasks used in these example applications has to be assigned to existing keycloak users. 

- `jdoe` : acts as the travel agent
- `alice` : acts as the visa officer

</details>

### CODE REVIEW

File: kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+KOGITO_VERSION=latest
```
Comment (dmartinol) [inline kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/.env]: Do we really need this file in the git repo?

**Review by dmartinol — COMMENTED** [review]
Fine for me, thanks!
@cristianonicolai could you please have a look?

File: kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/.env (hunk 1 above)
Comment (cristianonicolai) [reply kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/.env]: this is supposed to be generated during the `startServices.sh` script.

File: kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/svg/flightBooking.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...aseline="alphabetic">StartProcess</text></g></g><g id="SequenceFlow_2" bpmn2nodeid="SequenceFlow_2"><g><path fill="none" stroke="#000000" paint-order="fill stroke markers" d=" M 172 143 L 231 143" stroke-miterlimit="10" stroke-dasharray=""/></g><g transform="matrix(1,0,0,1,172,143)"/><g transform="matrix(6.123233995736766e-17,1,-1,6.123233995736766e-17,246,138)"><path fill="#000000" stroke="#000000" paint-order="fill stroke markers" d=" M 10 15 L 0 15 L 5 0 Z" stroke-miterlimit="10" stroke-dasharray=""/></g><g transform="matrix(1,0,0,1,172,133)"/></g><g id="SequenceFlow_1" bpmn2nodeid="SequenceFlow_1"><g><path fill="none" stroke="#000000" paint-order="fill stroke markers" d=" M 356 143 L 415 143" stroke-miterlimit="10" stroke-dasharray=""/></g><g transform="matrix(1,0,0,1,356,143)"/><g transform="matrix(6.123233995736766e-17,1,-1,6.123233995736766e-17,430,138)"><path fill="#000000" stroke="#000000" paint-order="fill stroke markers" d=" M 10 15 L 0 15 L 5 0 Z" stroke-miterlimit="10" stroke-dasharray=""/></g><g transform="matrix(1,0,0,1,356,133)"/></g><g transform="matrix(1,0,0,1,430,115)"/><g transform="matrix(1,0,0,1,246,118)"/><g transform="matrix(1,0,0,1,116,115)"/></g></g></svg>
```
Comment (cristianonicolai) [inline kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/svg/flightBooking.svg]: these are supposed to be copied from the target folder when executing the `startServices.sh` script.

File: kogito-quarkus-examples/kogito-travel-agency/extended/visas/src/main/resources/org/acme/travels/visa-approval-rules.drl [hunk 3]
```diff
@@ -16,20 +16,22 @@
...
 
 import org.acme.travels.VisaApplication;
-
+import org.acme.travels.VisaResolution;
 
 rule "Visit Australia with e-visa up to 90 days" ruleflow-group "visa-approvals"
 
 when
-    $application: VisaApplication(duration <= 90, country == "Australia")
+    $application: VisaApplication(duration <= 90, country == "Australia") && $resolution : VisaResolution()
```
Comment (cristianonicolai) [inline kogito-quarkus-examples/kogito-travel-agency/extended/visas/src/main/resources/org/acme/travels/visa-approval-rules.drl]:
```suggestion
    $application: VisaApplication(duration <= 90, country == "Australia")
    $resolution : VisaResolution()
```

File: kogito-quarkus-examples/kogito-travel-agency/extended/visas/src/main/resources/org/acme/travels/visa-approval-rules.drl [hunk 4]
```diff
@@ -16,20 +16,22 @@
...
-    $application.setApproved(true);     
+    $resolution.setApproved(true);
+    $resolution.setReason("Automatic visa approval");
 end
 
 rule "Visit US with e-visa up to 15 days" ruleflow-group "visa-approvals"
 
 when
-    $application: VisaApplication(duration <= 15, country == "US")
+    $application: VisaApplication(duration <= 15, country == "US") && $resolution : VisaResolution()
```
Comment (cristianonicolai) [inline kogito-quarkus-examples/kogito-travel-agency/extended/visas/src/main/resources/org/acme/travels/visa-approval-rules.drl]:
```suggestion
    $application: VisaApplication(duration <= 15, country == "US")
    $resolution : VisaResolution()
```

File: kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/.env (hunk 1 above)
Comment (JudeNiroshan) [reply kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/.env]: 815bada21c7902e37cd1fe98fc3961e2839b46b3

File: kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/svg/flightBooking.svg (hunk 2 above)
Comment (JudeNiroshan) [reply kogito-quarkus-examples/kogito-travel-agency/extended/docker-compose/svg/flightBooking.svg]: 815bada21c7902e37cd1fe98fc3961e2839b46b3

**Review by cristianonicolai — APPROVED** [review]

**Review by dmartinol — APPROVED** [review]

**Comment by dmartinol** [comment]
@cristianonicolai
Can anyone with write privileges to the repo merge the PR?
Thanks,
Daniele

**Comment by JudeNiroshan** [comment]
**Please make sure to squash and merge with the following message!**

> Kogito-6694 : Assign user tasks to jdoe and alice in extended/travels & extended/visas

**Pull request merged**